### PR TITLE
[VM] fix state test

### PR DESCRIPTION
**Pull request by jochem-brouwer**
CI on master is failing, this PR fixes that problem.

### CODE REVIEW

File: packages/vm/package.json [hunk 1]
```diff
@@ -14,12 +14,12 @@
     "coverage:test": "tape './tests/api/**/*.js' ./tests/tester.js --state --dist",
     "docs:build": "typedoc --options typedoc.js",
     "test:state": "ts-node ./tests/tester --state",
-    "test:state:allForks": "echo {'Homestead','TangerineWhistle','SpuriousDragon','Byzantium','Constantinople','Petersburg','Istanbul','MuirGlacier'} | xargs -n1 | xargs -I v1 npm run test:state -- --fork=v1",
-    "test:state:selectedForks": "echo {'Homestead','TangerineWhistle','SpuriousDragon','Petersburg',} | xargs -n1 | xargs -I v1 npm run test:state -- --fork=v1",
+    "test:state:allForks": "echo 'Homestead TangerineWhistle SpuriousDragon Byzantium Constantinople Petersburg Istanbul MuirGlacier' | xargs -n1 | xargs -I v1 npm run test:state -- --fork=v1",
+    "test:state:selectedForks": "echo 'Homestead TangerineWhistle SpuriousDragon Petersburg' | xargs -n1 | xargs -I v1 npm run test:state -- --fork=v1",
```
Comment (evertonfraga) [inline packages/vm/package.json:18]:
```suggestion
    "test:state:selectedForks": "echo 'Homestead TangerineWhistle SpuriousDragon Petersburg' | xargs -n1 -I v1 npm run test:state -- --fork=v1",
```

File: packages/vm/package.json [hunk 2]
```diff
@@ -14,12 +14,12 @@
...
     "test:state:slow": "npm run test:state -- --runSkipped=slow",
     "test:buildIntegrity": "npm run test:state -- --test='stackOverflow'",
     "test:blockchain": "node -r ts-node/register --stack-size=1500 ./tests/tester --blockchain",
-    "test:blockchain:allForks": "echo {'Homestead','TangerineWhistle','SpuriousDragon','Byzantium','Constantinople','Petersburg','Istanbul','MuirGlacier'} | xargs -n1 | xargs -I v1 node -r ts-node/register --stack-size=1500 ./tests/tester --blockchain --fork=v1",
+    "test:blockchain:allForks": "echo 'Homestead TangerineWhistle SpuriousDragon Byzantium Constantinople Petersburg Istanbul MuirGlacier' | xargs -n1 | xargs -I v1 node -r ts-node/register --stack-size=1500 ./tests/tester --blockchain --fork=v1",
```
Comment (evertonfraga) [inline packages/vm/package.json:22]: yeah, the JSON notation is really not mandatory. good catch.

File: packages/vm/package.json (hunk 1 above)
Comment (jochem-brouwer) [reply packages/vm/package.json:18]: This does not seem to work; https://github.com/ethereumjs/ethereumjs-vm/pull/831/checks?check_run_id=980780090

**Comment by jochem-brouwer** [comment]
Squashed commits. Note that the changes reflected in this PR are not tested in this PR (they will run on `master` though). However they have been tested; see this one: https://github.com/ethereumjs/ethereumjs-vm/actions/runs/205832577

Ready for re-review @evertonfraga

**Pull request merged**